### ISSUE TITLE
Linting fails under tflint 0.12.0

### ISSUE THREAD

**Issue by tillig** [hashicorp/vscode-terraform#229]
Given a very simple Terraform config file (not the real GUIDs and such):

```tf
provider "azurerm" {
  version         = "=1.35.0"
  subscription_id = "1fb2265c-7cd1-4d4f-bc67-61d7799765e8"
  tenant_id       = "f6586130-34a9-4ba8-917d-f78e55f72733"
}

resource "azurerm_resource_group" "kubernetes" {
  name     = "myrg"
  location = "West US"
}
```

Running linting (manually, since it doesn't seem to auto-run?) yields the following error in the output window:

```
2019-10-08 18:38:54:080 [INFO ] <cmd>lint: undefined issues
2019-10-08 18:38:54:081 [ERROR] <cmd>lint: Linting failed for c:\Users\tillig\Desktop\Terraform TypeError: issues.forEach is not a function
```

I believe this will be fixed by PR #227

However, if I add something to my config file [that would be found by linting, like an unpinned module](https://github.com/wata727/tflint/blob/master/docs/rules/terraform_module_pinned_source.md)...

```tf
provider "azurerm" {
  version         = "=1.35.0"
  subscription_id = "be62f057-87be-48e5-9c5f-c6fc74ac9d19"
  tenant_id       = "a172c77d-91cf-47f0-a813-30aee710c0dc"
}

resource "azurerm_resource_group" "kubernetes" {
  name     = "tillig-k8s"
  location = "West US"
}

module "unpinned" {
  source = "git://hashicorp.com/consul.git"
}
```

Running linting yields an error:

```
2019-10-08 18:44:26:069 [ERROR] <cmd>lint: Linting failed for c:\Users\tillig\Desktop\Terraform Error: Command failed: tflint --format json
```

I am guessing that when the current `tflint` finds errors it doesn't exit with 0, it instead appears to exit with 3, so [when linting sees any error, it doesn't even try to parse the results](https://github.com/mauve/vscode-terraform/blob/acbf3bde0854a4176d69ff444c29b8fe7ad526f3/src/commands/lint.ts#L101).

It appears you can run `tflint /force` to require it to always exit 0 even if issues are found. [Should that be added to the command line parameters?](https://github.com/mauve/vscode-terraform/blob/acbf3bde0854a4176d69ff444c29b8fe7ad526f3/src/commands/lint.ts#L90)

**Comment by tillig** [comment]
If #227 can be pulled in, I'd be happy to add `--force` to the command line args in a separate PR.

**Comment by jeffb4** [comment]
Oh thanks for this. My local tflint was `0.11.2` so I didn't see the behavior change.

**Comment by skupjoe** [comment]
I'm experiencing the same `Terraform TypeError: issues.forEach is not a function` on OSX for tflint version all the way down to `0.10.1`, where it finally passed successfully.

**Comment by mauve** [comment]
instead of adding `--force` it would be better just handle the error codes as specified by `tflint`: https://github.com/wata727/tflint/blob/master/README.md#exit-statuses

**Comment by mauve** [comment]
happy to do a release if we just fix that thing aswell

**Comment by tillig** [comment]
@jeffb4 Are you able to do this update or should I fork and submit a new version of the PR? I'd love to see tflint working.

**Comment by jeffb4** [comment]
@tillig I'm unable to do this update, sorry for the delay in response

**Comment by tillig** [comment]
Fair enough. I'll do my best to tackle it when I get a little breathing room.

**Comment by fmunteanu** [comment]
I'm experiencing the same error with tflint 0.13.4.

**Comment by tillig** [comment]
[PR submitted](https://github.com/mauve/vscode-terraform/pull/274) to handle exit codes and the new output format, which is a much deeper structure than it used to be.

**Comment by aaronsteers** [comment]
I'm experiencing this as well with terraform 0.12 - is there nothing which can be done as a workaround to get tflint working?

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
